translations/validators.bg.php

```diff
@@ -1,5 +1,5 @@
 <?php
 
 return [
-    'Should contains only letters, numbers, whitespace, dash and underscore' => 'Трябва да съдържа само букви, числа, дире и долна черта',
+    'Should contains only letters, numbers, whitespace, dash and underscore' => 'Трябва да съдържа само букви, числа, тире и долна черта',
```

Comment:
Please remove **whitespace** to match with translation

Reply:
fixed